### ISSUE TITLE
Collision between text and spinner icon

### ISSUE THREAD

**Issue by amarchino** [Napster2210/ngx-spinner#136]
Greetings.
I am using the 8.1.0 version of the library tor an Angular 8 project.
During a suite of accessibility tests, it was reported that, with great zoom of the page, the spinner and the underlying text collide with each other.
I tried to replicate the test in the demo page, and even on desktop, at sufficient levels of zoom (around 200% on both Firefox and Chromium-based browsers) the collision may be seen.

I suppose this may be due to the absolute positioning of the icon and the text as two separate entities.
Is there a way we might mitigate the problem? I tried via some CSS media queries, but it didn't work correctly on most zoom levels or screen widths.

Thanks!

**Comment by rspl-yuvraj** [comment]
Hi @amarchino ,
Do you want the spinner bigger when you zoom around 200%? or you can set spinner size `default` while you zoom to 200%

**Comment by rspl-yuvraj** [comment]
Take a reference: https://css-tricks.com/can-javascript-detect-the-browsers-zoom-level/
And based on your zoom level you can configure the spinner or override the style of spinner.

**Comment by amarchino** [comment]
Sorry for the late reply.
I tried to apply the given instructions, trying to adjust the spinner size at some zoom break points.
The result is only kind of encouraging (mainly due to the lack of high level of support for zoom levels in browsers, given that the currently accessible API is not very flexible nor powerful): by changing the size, the collision was put of until approximately 450% zoom.
After that breakpoint, nothing I did prevented the collision, save for manually changing the position of the elements, which in turn looked incorrect at lower level of zooms.

Given that this level of zoom is fairly uncommon, I can safely consider the issue handleable with the bit of "offside code" handling the zoom-related sizing manually.

Thanks for the attention given to this issue!
